esrally/metrics.py

```diff
@@ -2007,6 +2028,11 @@ def op_metrics(op_item, key, single_value=False):
             elif metric.startswith("total_transform_") and value is not None:
                 for item in value:
                     all_results.append({"id": item["id"], "name": metric, "value": {"single": item["mean"]}})
+            elif metric.startswith("field_disk_usage_") and value is not None:
+                for item in value:
+                    index = item["index"]
+                    field = item["field"]
+                    all_results.append({"id": f"{index} {field}", "name": metric, "value": {"single": item["value"]}})
```

Comment:
I think this is where the problem with the elasticsearch based metric store shows up.